zerver/tests/test_auth_backends.py

```diff
@@ -791,36 +784,55 @@ def test_social_auth_registration(self) -> None:
         result = self.client_get(result.url)
 
         self.assertEqual(result.status_code, 302)
-        confirmation = Confirmation.objects.all().first()
+        confirmation = Confirmation.objects.all().last()
         confirmation_key = confirmation.confirmation_key
         self.assertIn('do_confirm/' + confirmation_key, result.url)
         result = self.client_get(result.url)
         self.assert_in_response('action="/accounts/register/"', result)
         data = {"from_confirmation": "1",
-                "full_name": name,
                 "key": confirmation_key}
         result = self.client_post('/accounts/register/', data)
-        self.assert_in_response("We just need you to do one last thing", result)
-
-        # Verify that the user is asked for name but not password
-        self.assert_not_in_success_response(['id_password'], result)
-        self.assert_in_success_response(['id_full_name'], result)
-
-        # Click confirm registration button.
-        result = self.client_post(
-            '/accounts/register/',
-            {'full_name': name,
-             'key': confirmation_key,
-             'terms': True})
+        if skip_registration_form is False:
```

Comment:
`is False` is an unusual way to write conditionals in Python; stylistically, we should just do `if not skip_registration_Form`.

Reply:
I really like that ``is False`` style :see_no_evil: But changed to the more usual ``not ...`` form now :slightly_smiling_face: